onboarding/scaffolding.py

```diff
@@ -192,6 +192,23 @@ def resolver(_, info: ResolveInfo):
         return graphene.Field(cls, resolver=resolver)
 
 
+def get_scaffolding_fields_from_form(form) -> Dict[str, Any]:
+    """
+    Returns the form's cleaned data in a format that is suitable
+    for updating scaffolding data.
+
+    Generally this is just the form's cleaned data. However, if the
+    form has a `to_scaffolding_keys` property that maps its field
+    names to scaffolding field names, that is used to rename fields
+    as needed.
```

Comment:
This is a new convention I added to improve DRY.  A form class can just define `to_scaffolding_keys` once and then other things can use that, instead of us having to repeat the renamings all over the place.